tests/lint_examples/minimal_working_example/.travis.yml

```diff
@@ -10,6 +10,8 @@ matrix:
   fast_finish: true
 
 before_install:
+  # PRs made to 'master' branch should always orginate from another repo or the 'dev' branch
```

Comment:
This comment is not really correct. PRs to master from another repo isn't ok either? Right? The code looks good though. Perhaps change to "PRs to master is only ok from dev branch"

Reply:
Correct, this is not correct. I'll update